config/webpack.config.dev.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
     // https://github.com/facebookincubator/create-react-app/issues/253
     fallback: paths.nodePaths,
     // These are the reasonable defaults supported by the Node ecosystem.
-    extensions: ['.js', '.json', ''],
+    extensions: ['.jsx', '.js', '.json', ''],
```

Comment:
I would prefer Node resolution mechanism to be respected before falling back to JSX.